pkg/process/checks/host_info_test.go

```diff
@@ -100,7 +100,7 @@ func TestGetHostnameFromCmd(t *testing.T) {
 }
 
 func TestResolveHostname(t *testing.T) {
-	if os.Getenv("CI") == "true" && runtime.GOOS == "darwin" && runtime.GOARCH == "amd64" {
+	if os.Getenv("CI") == "true" && runtime.GOOS == "darwin" && (runtime.GOARCH == "amd64" || runtime.GOARCH == "arm64") {
```

Comment:
I think we may as well drop the part about the architecture, I can't think of a good reason to keep it:
```suggestion
	if os.Getenv("CI") == "true" && runtime.GOOS == "darwin" {
```

Same for the rest of the occurrences (I won't add extra comments for those).